security.tex

```diff
@@ -0,0 +1,251 @@
+\chapter{Debug Security}
+
+{\bf This part of the spec needs work before it's ready to be implemented,
+which is why it's in the appendix. It's left here to give a rough idea of some
+of the issues to consider.}
+
+\section{Rationale}
+
+Security is imperative within any computing ecosystem, but especially in the context of a debugging model for any given processing architecture. The RISC-V debugging architecture has been designed for flexibility, ease, and modularity. As such, ensuring security can be enforced as much as is reasonably possible is a critical part of the RISC-V debug specification. The following considerations illuminate this position:
+\begin{itemize}
```

Comment:
(NB I wrote a lot of these comments before you gave your presentation, I just never hit 'SAVE' apparently)